cmd/root.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+    "fmt"
+    "os"
+
+    homedir "github.com/mitchellh/go-homedir"
+    "github.com/spf13/cobra"
+    "github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+    Use:   "chesstime-tool",
+    Short: "A brief description of your application",
+    Long: `A longer description that spans multiple lines and likely contains
+examples and usage of using your application. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+    // Uncomment the following line if your bare application
+    // has an action associated with it:
+    //    Run: func(cmd *cobra.Command, args []string) { },
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+    if err := rootCmd.Execute(); err != nil {
+        fmt.Println(err)
+        os.Exit(1)
```

Comment:
could use a panic() instead, look up "exiting golang application"

Reply:
Oh this was generated with cobra.